src/CHIPBindings.cc

```diff
@@ -1305,7 +1306,6 @@ hipError_t hipStreamWaitEvent(hipStream_t Stream, hipEvent_t Event,
                               unsigned int Flags) {
   CHIP_TRY
   CHIPInitialize();
-  NULLCHECK(Event);
```

Comment:
Why remove this check?

Reply:
there is another nullcheck on event, a few lines below